apisix/ssl/router/radixtree_sni.lua

```diff
@@ -258,7 +258,8 @@ function _M.set(matched_ssl, sni)
             local reject_in_handshake =
                 (ngx.config.subsystem == "stream") or
                 (matched_ssl.value.client.skip_mtls_uri_regex == nil)
-            local ok, err = ngx_ssl.verify_client(parsed_cert, depth,
+            -- TODO: support passing `trusted_certs` (3rd arg, keep it nil for now)
+            local ok, err = ngx_ssl.verify_client(parsed_cert, depth, nil,
```

Comment:
the verify client API has changed and it expects 4 parameters now, the 3rd parameter is `trusted_certs`, we can implement the support for trusted certs later.